tests/utils/iotrace.py

```diff
@@ -163,7 +163,7 @@ def get_traces_list(prefix: str = None, shortcut: bool = False) -> list:
             command += (' -p ' if shortcut else ' --prefix ') + f'{prefix}'
         output = TestRun.executor.run_expect_success(command)
 
-        return parse_json(output.stdout)
+        return parse_json(output.stdout)[0]["trace"]
```

Comment:
Please, update the use of this method, especially in the next one.

Reply:
Fixed!